server/plugin/plugin.go

```diff
@@ -81,6 +87,48 @@ func (p *MatterpollPlugin) checkServerVersion() error {
 	return nil
 }
 
+// ensureBotAccount checks if the bot account exists and creates him if doesn't
+func (p *MatterpollPlugin) ensureBotAccount() error {
+	botUserID, err := p.Store.Bot().GetID()
+	if err != nil {
+		return errors.Wrap(err, "failed to get bot user from store")
+	}
```

Comment:
What should happen, if an admin disables the bot? Just enable it back?

Reply:
Maybe this isn't a problem. I've tested and you can still create messages, if the bot account is disabled.

Reply:
It sounds good, but who does create a poll if bot is disabled?